### DISCUSSION TITLE
C++ overload functions not display?

### DISCUSSION THREAD

**Discussion by karriszou** [company-mode/company-mode#1301] [Q&A]
I want to display all overload function in company tooltip, but only the lastest overload function is  displayed in candidates.

    How can i fix this ?

Thanks!

**Comment by yugaego** [comment]
A similar question has been discussed in the [How to cycle overloads](https://github.com/company-mode/company-mode/issues/1008) issue.

To investigate your exact case, what's the output of the `M-x company-diag` command?

**Reply by dgutov** [reply]
Yes, to sum up: this depends on the backend, whether to show all overloads as separate completions (all "classic" company backends have done this), or to group them. LSP apparently prefer the latter, but this can also depend on the server, and its configuration.

So if you are using a particular language server, you can read its docs and ask for this on its issue tracker.
